### PR TITLE
30 create footer component

### LINKED ISSUE #58: 30 - Create footer component
[FOOTER](https://www.figma.com/file/MpfNES0KuBUkkd0Qk4EyOc/HYF-Connect-Wireframe?node-id=463%3A319)

- [x] Create a div container class:
- [x] Create a paragraph linked to Privacy Policy / terms and conditions page
- [x] Insert the HYF-Connect logo
- [x] Insert the copyright info
- [x] Insert 3 social media icons with their links
- [x] Insert a paragraph

### PR DESCRIPTION
**Pull request by RosaMoran**
hey Llamas I'm still working but I have some problem with css, and i was trying to fix but nop, tomorrow I'll push branch `contact us` for tonight I'll go to sleep.

by the way ``` have a look```  file ```acces.log.js```

I din't touche nothing from there

### CODE REVIEW

**Comment by dirk-jacobs** [comment]
There are a number of problems still. We don't have a page to test the footer.

- The reference from eg about-us page to the header and footer component is incorrect (wrong filename)
- The reference to the style file is incorrect.

Is it possible to correct this ?

File: client/pages/about-us/about-us.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import HeaderComponent from "../../components/header-component.js";
-import FooterComponent from "../../components/footer-component.js";
+import HeaderComponent from "../../components/header.js";
+import FooterComponent from "../../components/footer.js";
```
Comment (dirk-jacobs) [inline client/pages/about-us/about-us.js]: Incorrectly changed. No idea why

File: package.json [hunk 2]
```diff
@@ -27,8 +27,12 @@
     "body-parser": "^1.19.0",
     "cors": "^2.8.5",
     "dotenv": "^10.0.0",
+<<<<<<< HEAD
+    "express": "^4.17.2",
+=======
     "express": "^4.17.1",
     "jsonwebtoken": "^8.5.1",
+>>>>>>> 0a7fd17471769049a67f564360e1804716e16062
     "morgan": "^1.10.0",
```
Comment (dirk-jacobs) [inline package.json]: PLease solve these issues

**Review by dirk-jacobs — CHANGES_REQUESTED** [review]
Please provide a 'demo'-page where we can test the footer

**Comment by dirk-jacobs** [comment]
Also, please, exclude the access.log

File: package.json (hunk 2 above)
Comment (RosaMoran) [reply package.json]: i solved

File: client/pages/about-us/about-us.js (hunk 1 above)
Comment (RosaMoran) [reply client/pages/about-us/about-us.js]: I saw you fixed and I fixed in footer and i added ``FooterComponent``  to homepage

**Comment by RosaMoran** [comment]
@dirk-jacobs  @rayanejsilva  and @Moamin-AbuEwaida @yoshimalaise 
 i fixed 
```client/pages/about-us/about-us.js```
```package.json```

i saw that you @dirk-jacobs fixed the path of pages  and add ```header-componen.js ``` and  ```footer-component.js```

  and footer is finished so you can see it please and it's fine you can approved and merged please

**Comment by Moamin-AbuEwaida** [comment]
Thanks @RosaMoran I have few comments on the footer
- the logo in the center of the footer styling needs modification
- the Privacy Policy & Terms and conditions  should be clickable <a> elements

**Review by Moamin-AbuEwaida — COMMENTED** [review]
shared some comments regarding the logo styling and the Privacy Policy | Terms and Condition to be <a> elements

**Review by dirk-jacobs — CHANGES_REQUESTED** [review]
package-lock.json is broken > run npm install to regenerate and checkin
pull from development again and get it running

**Comment by RosaMoran** [comment]
@Moamin-AbuEwaida  and @dirk-jacobs  i made that you suggested buddies, it's ready to be merged

**Review by rayanejsilva — COMMENTED** [review]
@RosaMoran you deleted sign-up page styling.

**Review by dirk-jacobs — DISMISSED** [review]
This version looks rather good to me.
I would merge it with development

**Comment by perezrei** [comment]
Observations:

1. The 3 social media icons are linked to GitHub, but we can fix that later.
2. I think we haven't discussed what our terms and conditions are, so...
3. The phrase "code along with us" is linked to GitHub too, I don't know if that was the idea.
4. Some minor alignments touches are needed but, again, can be fixed later

**Review by perezrei — APPROVED** [review]

**Review by perezrei — DISMISSED** [review]

**Review by perezrei — APPROVED** [review]

**Review by rayanejsilva — APPROVED** [review]

**Pull request merged**